### DISCUSSION TITLE
Make an installer

### DISCUSSION THREAD

**Discussion by Thatsnice1938** [ntsc-rs/ntsc-rs#162] [Ideas]
I think you should :)

**Comment by Degamisu** [comment]
An installer for which part? The OFX or standalone? Either way I wouldn't see a point as they can easily be set up. Thats my opinion though, I could be wrong. 😃

**Comment by sakhalif456** [comment]
i dont want that 😡😡

**Reply by valadaptive** [reply]
Please stop spamming the Discussions page.
